packages/pn-pa-webapp/cypress/e2e/NewNotification.cy.ts

```diff
@@ -292,4 +293,220 @@ describe("Notifications New Notification", () => {
+    
+    // Fill step 1
+    cy.fillPreliminaryInfo({
+      paProtocolNumber: 'prot-123456789',
+      subject: 'Cypress Test - Create new notification',
+      abstract: 'Testing as administrator and without user groups',
+      taxonomyCode: '012345X',
+      communicationType: 'Model_890',
+      paymentMethod: 'pagoPA'
+    });
+
+    cy.get('#group').should('be.disabled');
+
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.fillRecipient({
+      position: 0,
+      data: {
+        ...cesare,
+        creditorTaxId: '77777777777',
+        noticeCode: '302001869076319100',
+      }
+    });
+    
+    cy.get('button[type="submit"]').should('be.enabled').click();
+    
+    // Fill step 3
+    cy.get('input[type="file"]').eq(0).selectFile(pdfTest1, { force: true });
+    cy.get('input[name="documents\.0\.name"]').type('pdf di Test 1');
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.wait('@preloadAttachments').its('response.statusCode').should('eq', 200);
+
+    // Fill step 4
+    cy.get('input[type="file"]').eq(0).selectFile(pdfTest2, { force: true });
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.wait('@saveNewNotification');
+
+    cy.contains('La notifica è stata correttamente creata');
+  });
+
+  it('Create new notification with groups and administrator role', () => {
+
+    cy.intercept(/groups/, { fixture: 'groups/groups' });
+    
+    // Fill step 1
+    cy.fillPreliminaryInfo({
+      paProtocolNumber: 'prot-123456789',
+      subject: 'Cypress Test - Create new notification',
+      abstract: 'Testing as administrator and with user groups',
+      taxonomyCode: '012345X',
+      communicationType: 'Model_890',
+      paymentMethod: 'pagoPA'
+    });
+
+    // The following validation need to be resolved with PN-2198
+    // cy.get('button[type="submit"]').should('be.disabled')
+
+    cy.get('#group').click();
+    cy.get('.MuiMenuItem-root').click();
+
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.fillRecipient({
+      position: 0,
+      data: {
+        ...cesare,
+        creditorTaxId: '77777777777',
+        noticeCode: '302001869076319100',
+      }
+    });
+    
+    cy.get('button[type="submit"]').should('be.enabled').click();
+    
+    // Fill step 3
+    cy.get('input[type="file"]').eq(0).selectFile(pdfTest1, { force: true });
+    cy.get('input[name="documents\.0\.name"]').type('pdf di Test 1');
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.wait('@preloadAttachments').its('response.statusCode').should('eq', 200);
+
+    // Fill step 4
+    cy.get('input[type="file"]').eq(0).selectFile(pdfTest2, { force: true });
+    cy.get('button[type="submit"]').should('be.enabled').click();
+
+    cy.wait('@saveNewNotification');
+
+    cy.contains('La notifica è stata correttamente creata');
+  });
+
+  it('Create new notification with no user groups and operator role', () => {
```

Comment:
Are you sure you are acting as an operator here and in the following test? It looks like you forgot to logout and login as operator or call setRole.